lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -263,39 +263,25 @@ def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
             'equalyz'   adapt the y and z axes to have equal aspect ratios.
             =========   ==================================================
 
-        adjustable : None or {'box', 'datalim'}, optional
-            If not *None*, this defines which parameter will be adjusted to
-            meet the required aspect. See `.set_adjustable` for further
-            details.
-
-        anchor : None or str or 2-tuple of float, optional
-            If not *None*, this defines where the Axes will be drawn if there
-            is extra space due to aspect constraints. The most common way to
-            specify the anchor are abbreviations of cardinal directions:
-
-            =====   =====================
-            value   description
-            =====   =====================
-            'C'     centered
-            'SW'    lower left corner
-            'S'     middle of bottom edge
-            'SE'    lower right corner
-            etc.
-            =====   =====================
+        ``adjustable`` : None or {'box', 'datalim'}, optional
```

Comment:
Why is this code style now?

Reply:
This was working fine before; I don't think the error you were seeing had to do with this line.